src/Response.php

```diff
@@ -140,7 +151,7 @@ public function getResult()
      * @param  mixed $error
      * @return self
      */
-    public function setError(Error $error = null)
+    public function setError(Error $error = null) : self
```

Comment:
If I remember correctly it will be enforced for LCS 2.0 - typehint for nullable parameters should be always with `?`. Can be addressed later with other CS changes.

(Also just to note: with LCS 2.0 we go with PSR-12 return type hint formatting, so no space before `:`).

Reply:
Nullable type declaration: I tend to add it, must have missed this one. Will provide the change shortly.

Same for no space before return type declaration colon.